tests/test_core.py

```diff
@@ -2394,8 +2401,8 @@ def test_dlfcn_i64(self):
       '''
     self.do_run(src, '|65830|')
 
-  @no_wasm # TODO: EM_ASM in shared wasm modules, stored inside the wasm somehow
   @needs_dlfcn
+  @no_wasm() # TODO: EM_ASM in shared wasm modules, stored inside the wasm somehow
```

Comment:
Because no_wasm takes a note argument, should we move these TODOs into note strings?

Reply:
good idea